code_challenges/code_wars/instance.py

```diff
@@ -0,0 +1,21 @@
+class Vehicle:
+    def __init__(self, seats, wheels, engine):
+        self.seats = seats
+        self.wheels = wheels
+        self.engine = engine
+
+
+class Planet:
+    def __init__(self, moon):
+        self.moon = moon
```

Comment:
These aren't really a part of the solution, but part of testing, so should live in the test file instead.